yarn.lock

```diff
@@ -5859,14 +5859,6 @@
   resolved "https://registry.yarnpkg.com/@strapi/icons/-/icons-1.2.7.tgz#35862d7f6a088f9a0016b1d81687294c77526d0d"
   integrity sha512-Uae7t/N4d+9zolgPFsbU4bq4+Y9zE90Lv1HKFKvF/3DFWX5W8Q1HRMN+sawdno7LXlHqav2TUXssJ18wwYfrRw==
 
-"@strapi/logger@4.4.6":
```

Comment:
Why did this file change?

Reply:
Because we're now using the 4.5.0 strapi packages, it has probably been forgotten when upgrading from 4.4.6 to 4.50